messages/ctrlMsg.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package mateROV;
+
+message CtrlMsg	{
+
+	required float x = 0.
```

Comment:
The numbers here are just ids for these fields, not initial values. As such they should be ints in increasing order starting from 1.
Eg.
```
required float x = 1
required float y = 2
required float z = 3
...
```